shared/js/bluetooth_helper.js

```diff
@@ -252,6 +268,18 @@ var BluetoothHelper = function() {
       });
     },
 
+    set ona2dpstatuschanged(callback) {
+      _ready(function() {
+        _adapter.ona2dpstatuschanged = callback;
+      });
+    },
+
+    set onrequestmediaplaystatus(callback) {
+      _ready(function() {
+        _adapter.onrequestmediaplaystatus = callback;
+      });
+    },
+
```

Comment:
The original author of the helper is @crh0716 . I have asked him about the scope for the helper before. It's only serving these methods for apps(callscreen, Bluetooth, and Music apps now). And these methods should be using really. It helps apps no longer to handle(get/reset) default adapter by themselves. During this transition(v1 -> v2), the helper is also a good role to handle different API version. The minute and complicated things is covered in this helper.